store/sqlstore/team_store.go

```diff
@@ -6,26 +6,22 @@ package sqlstore
 import (
 	"database/sql"
 	"fmt"
+	"github.com/mattermost/mattermost-server/einterfaces"
 	"net/http"
 	"strings"
 
 	sq "github.com/Masterminds/squirrel"
 	"github.com/mattermost/gorp"
-	"github.com/mattermost/mattermost-server/einterfaces"
 	"github.com/mattermost/mattermost-server/model"
 	"github.com/mattermost/mattermost-server/store"
-	"github.com/mattermost/mattermost-server/utils"
 )
 
 const (
-	TEAM_MEMBER_EXISTS_ERROR         = "store.sql_team.save_member.exists.app_error"
```

Comment:
Why are you removing this constant here?

Reply:
That was a mistake, let me check. I could be a merge error maybe.

Reply:
This is strange, I don't have this change even, the line is still there.

https://github.com/mattermost/mattermost-server/blob/751f308ec5238895c0621bbffaa98ab3f85a0884/store/sqlstore/team_store.go

I think this happened because of the rearrangement of the imports (that was non intentional too) so I'll fix it

Reply:
I removed `"github.com/mattermost/mattermost-server/utils"` only